js-html-games/coinScript.js

```diff
@@ -0,0 +1,91 @@
+const slider = document.getElementById('slider');
+const counter = [0, 0];
+
+function createEverything() {
+
+    const imgList = document.createElement('ul');
+    const img = document.createElement('img');
+    const result = document.createElement('p');
+    
+    result.innerHTML = "Results: _______";
+    img.setAttribute('src', "./animation/frame_00_delay-0.07s.gif");
+    slider.appendChild(imgList);
+    imgList.appendChild(img);
+    imgList.appendChild(result);
+
+    function setImgNum(num, target) {
+        if (num < 10) {
+            target.setAttribute('src', `./animation/frame_0${num}_delay-0.07s.gif`);
+        } 
+        else {
+            target.setAttribute('src', `./animation/frame_${num}_delay-0.07s.gif`);
+        }
+    }
+    
+    const SPEED = 10;
+    
+    function run(n, start, end) {
+        let count = start;
+        let lastCount = 0;
+        function next(last) {
+            if (count < last) {
+                setImgNum(count, img)
+                // console.log("count", count);
+                count++;
+            }
+            else if (count === last && lastCount !== n) {
+                // console.log("lastCount is ", lastCount)
+                count = 0;
+                lastCount++;
+            }
+        }
+        setInterval(function() {next(end)}, SPEED);
+    }
+    
+    function genResult() {
+        const flipResult = Math.random();
+        if (flipResult < 0.5) {
+            console.log("heads");
+            return "heads";
+        } 
+        else {
+            console.log("tails");
+            return "tails";
+        }
+    }
+    
+    switch (genResult()) {
+        case "heads":
+            run(1, 0, 9);
+            counter[0]++;
+            result.innerHTML = `Results: <strong>heads</strong>`;
+            break;
+        case "tails":
+            run(1, 9, 17);
```

Comment:
recommend moving the exact numbers out (1, 0,9; 1, 9,17) and out into specific constant variables. Here's why: 
- https://softwareengineering.stackexchange.com/questions/67982/is-it-ever-a-good-idea-to-hardcode-values-into-our-applications
- http://wiki.c2.com/?MagicNumber


so for me as a reader/code reviewer, skimming thru the code i'm not sure why we are using 0,9 and 9,17 values specifically. would be helpful to have a descriptive variable for it